README.md

```diff
@@ -34,7 +34,9 @@
     - [Depth Estimation (3D Reconstruction)](#depth-estimation)
         - [Monocular](#depth-mono)
         - [Stereo](#depth-stereo)
+        - [Stereo Depth Estimation using Structured Light](#depth-stereo-active)
         - [Stereoscopic panoramic imaging](#depth-stereo-pano)
+        - [LiDAR Guided Stereo Depth Estimation](#depth-stereo-fusion)
```

Comment:
Thanks. Could we have a subsection called "Events and LiDAR", which collects works on depth estimation using the combination of both modalities (monocular or stereo)?
Then, we could write here at least two entries:
- Brebion et al SCIA 2023  (move this entry from the current monocular section)
- Bartolomei et al ECCV 2024.